### ISSUE TITLE
plugin Toolbar zoomOut somtimes not work

### ISSUE THREAD

**Issue by xiaofengqqcom123** [antvis/G6#2539]
version: @antv/g6@3.8.1
path: .../node_modules/@antv/g6/lib/plugins/toolBar/index.js
```
  ToolBar.prototype.handleDefaultOperator = function (code, graph) {
	...
   case 'zoomOut':
        {
          var ratioOut = 1 + 0.05 * 5;

          if (ratioOut * currentZoom > 5) {
            return;
          }

          graph.zoomTo(currentZoom * 1.1);
          break;
        }

      case 'zoomIn':
        {
          var ratioIn = 1 - 0.05 * 5;

          if (ratioIn * currentZoom < 0.3) {
            return;
          }

          graph.zoomTo(currentZoom * 0.9);
          break;
        }
	...
  }
```
question：why declare variable ratioOut、 ratioIn？ 

I think
```
  ToolBar.prototype.handleDefaultOperator = function (code, graph) {
	...
   case 'zoomOut':
        {
          var ratioOut = 1 + 0.05 * 5;

          if (ratioOut * currentZoom > 5) {
            return;
          }

          graph.zoomTo(currentZoom * ratioOut);
          break;
        }

      case 'zoomIn':
        {
          var ratioIn = 1 - 0.05 * 5;

          if (ratioIn * currentZoom < 0.3) {
            return;
          }

          graph.zoomTo(currentZoom * ratioIn);
          break;
        }
	...
  }
```

**Comment by Yanyan-Wang** [comment]
You're right. What's more, the `ratioOut` should be `1 / (1 - DELTA * sensitivity)` and the `ratioIn` should be `1 - DELTA * sensitivity`, corresponding to the definition in `zoom-canvas` behavior: https://github.com/antvis/G6/blob/33482dcd59f6cbe534812649f24f45f12b564da3/packages/pc/src/behavior/zoom-canvas.ts#L150

It should be fixed in next small version, and we also welcome Pull Request from the community.

**Comment by Yanyan-Wang** [comment]
fixed by v4.1.3
